bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -184,21 +241,19 @@ def __init__(
         self._enforce_dtype = enforce_dtype
         self._shape = shape
         self._enforce_shape = enforce_shape
-        self._mime_type_request = "application/octet-stream" if from_parquet else "application/json"
-        self._mime_type_response = "application/octet-stream" if to_parquet else "application/json"
-        self._from_parquet = from_parquet
-        self._to_parquet = to_parquet
+        self._default_format = SerializationFormat[default_format.upper()]
 
     def openapi_schema_type(self) -> t.Dict[str, t.Any]:
         return _schema_type(self._dtype)
 
     def openapi_request_schema(self) -> t.Dict[str, t.Any]:
         """Returns OpenAPI schema for incoming requests"""
-        return {self._mime_type_request: {"schema": self.openapi_schema_type()}}
+        return {f"request.headers[\"content-type\"] (default: {self._default_format.mime_type})":
+                    {"schema": self.openapi_schema_type()}}
```

Comment:
```suggestion
        return {self._default_format.mime_type: {"schema": self.openapi_schema_type()}}
```

Reply:
Are we ok with using `_default_format.mime_type` here even though it can be overridden by the request headers["content-type"]?

Reply:
I think so.